server/bin/pbench-audit-server.py

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env python3
+# -*- mode: python -*-
+
+import os
+import sys
+import glob
+import re
+import errno
+import hashlib
+import shutil
+import tempfile
+import tarfile
+from enum import Enum
+from argparse import ArgumentParser
+from s3backup import S3Config, Entry
+
+from pbench import PbenchConfig, BadConfig, get_pbench_logger, md5sum
+from pbench.report import Report
+
+
+_NAME_ = "pbench-audit-server"
+
+def verify_subdirs(config, controller, cnt):
+
+    linkdirs = sorted(config.LINKDIRS.split(' '))
+
+    states = glob.iglob(os.path.join(controller,'*'))
```

Comment:
Seems like we might have more that just state directories from the `iglob` generator.  Can we use a name like `dirents`, or `directory_entires`?